Ginger/GingerPluginCore/ActionsLib/IUIElementAction.cs

```diff
@@ -20,7 +20,7 @@
 namespace Amdocs.Ginger.Plugin.Core
 {
 
```

Comment:
changes to IUIElementAction should be undo, right?

Reply:
no it is required, since the enums used in this was conflicting with the enums in the "SeleniumDriver"